tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -560,6 +560,49 @@ def test_pf_load_plugin():
     assert np.allclose(circ_times_area, ds["circ"] * ds["area_um"])
 
 
+@pytest.mark.filterwarnings(
+    "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
+def test_pf_load_non_scalar_plugin_data():
+    """Test loading non-scalar plugin data format"""
+    ds = dclab.new_dataset(retrieve_data("fmt-hdf5_fl_2018.zip"))
+    info = example_plugin_info_non_scalar_feature()
+    PlugInFeature("image_gauss_filter", info)
+
+    # Accessing non-scalar plugin data without prior saving and storing data in
+    # HDF5-format should return already computed data as np.ndarray
+    assert isinstance(ds["image_gauss_filter"], np.ndarray)
+
+    # Exporting the rtdc-file including the plugin-feature and then reloading
+    # it should load the non-scalar plugin data as h5py.Dataset
+    with tempfile.TemporaryDirectory() as temp_dir:
+        pdir = pathlib.Path(temp_dir)
+        pfile = pdir / "tmp.rtdc"
+        features = ds.features + ["image_gauss_filter"]
+        ds.export.hdf5(pfile, features=features)
+        ds2 = dclab.new_dataset(pfile)
```

Comment:
Windows tests (on Appveyor) fail, I think due to cleanup issues of TemporaryDirectory. Do you have an "ignore_error_on_cleanup" or so option for that?

Otherwise, maybe using a context manager for the new dataset `with dclab.new_dataset(pfile) as ds2` might work.

Reply:
I did not find anything about dealing with the cleanup-error, but will try your second approach. Thanks for the tips!

Reply:
Ok, so I tried the context manager for `ds2` and it did not work. It lead to a failing appveyor with no informative error messages and a very generic `failed with exit code 1`. So I looked into your first approach again and some other guy had a very similar issue:
https://github.com/appveyor/ci/issues/2547
I will look into that.

Reply:
OK, I'm going to merge anyway. Maybe it has something to do with builds for PRs on Appveyor.